src/test/java/org/opentripplanner/middleware/triptracker/ManageLegTraversalTest.java

```diff
@@ -667,6 +703,11 @@ public TraceData(Coordinates position, String expectedInstruction, String messag
             this(position, expectedInstruction, false, message);
         }
 
+        public TraceData(Coordinates position, String expectedInstruction, Instant instant, String message) {
```

Comment:
The TraceData class could now benefit from being refactored to use the builder pattern and being moved into it's own class. Perhaps out of scope, will let you decide.

Reply:
Updated in https://github.com/ibi-group/otp-middleware/pull/304/commits/b99ecce0f968731e8a01301245efab55b709872f.